dot/core/service_integration_test.go

```diff
@@ -423,52 +358,45 @@ func TestMaintainTransactionPool_EmptyBlock(t *testing.T) {
 	keyring, err := keystore.NewSr25519Keyring()
 	require.NoError(t, err)
 	alicePub := common.MustHexToBytes(keyring.Alice().Public().Hex())
-	encExt, runtimeInstance := generateTestValidRemarkTxns(t, alicePub, accountInfo)
-	cfg := &Config{
-		Runtime: runtimeInstance,
-	}
+	genesisFilePath, err := utils.GetGssmrGenesisRawPath()
+	require.NoError(t, err)
 
 	ctrl := gomock.NewController(t)
-	telemetryMock := NewMockClient(ctrl)
-	telemetryMock.EXPECT().SendMessage(gomock.Any()).AnyTimes()
+	service, encExt := createTestService(t, genesisFilePath, alicePub, accountInfo, ctrl)
 
-	transactionState := state.NewTransactionState(telemetryMock)
 	tx := &transaction.ValidTransaction{
 		Extrinsic: types.Extrinsic(encExt),
 		Validity:  &transaction.Validity{Priority: 1},
 	}
-	_ = transactionState.AddToPool(tx)
-
-	service := NewTestService(t, cfg)
-	service.transactionState = transactionState
+	_ = service.transactionState.AddToPool(tx)
 
 	// provides is a list of transaction hashes that depend on this tx, see:
 	// https://github.com/paritytech/substrate/blob/5420de3face1349a97eb954ae71c5b0b940c31de/core/sr-primitives/src/transaction_validity.rs#L195
 	provides := common.MustHexToBytes("0xd43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d00000000")
 	txnValidity := &transaction.Validity{
-		Priority:  39325240425794630,
+		Priority:  36074,
```

Comment:
Why did we have to change this?

Reply:
You know I am not entirely sure why it changed (since the runtime sets it), but since our runtimes as well as the transaction being tested has changed I am not surprised that this needed to be changed. However, if you find this suspicious let me know